### DISCUSSION TITLE
Clip Skip Setting when training

### DISCUSSION THREAD

**Discussion by jsbach-jung** [d8ahazard/sd_dreambooth_extension#604] [Q&A] [answered]
Hi, I've searched extensively about this, and plan to test it when the holiday craze is over. 

Are the following statements true?

- If clip skip is set to 1 in your Automatic UI Settings, at the time you train your model, it will target Clip Skip 1, much like SD 1.5
- If clip skip is set to 2 in your Automatic UI Settings, it will target the training on Clip Skip 2, like for NAI

I noticed this recently when I fine tuned a model (a mixed model from Civitai, similar to HassanBlend) of a person and noticed it produced "correct" results only on Clip Skip 2, and I realized I had left it on 2 in UI Settings while I had been experimenting with a NAI-based model right before training.

I was surprised because I'd mixed onto that mixed model before and it was trained on to Clip 1, but now I was only seeing results on Clip 2.

If this is not correct, is there a way we can "target" a particular Clip when we train, is it available in the parameters? Thanks.

**Comment by leppie** [comment, marked as answer]
> Are the following statements true?

I believe both are true when training the text encoder. 

Relevant code is here:

https://github.com/d8ahazard/sd_dreambooth_extension/blob/633ca33e8ca3d55a5a09a98b3ea6975bd8ffec29/dreambooth/finetune_utils.py#L211-L217
